behavior/MouseTracking/FindTrials/mouse_path_plotter.m

```diff
@@ -0,0 +1,42 @@
+function mouse_path_plotter( movie, centroids_mat)
+% Plots the path of the mouse during each trial based on position data and
+% trial frames extracted from behavior movie
+% 
+% Input:
+%     movie: Behavior video
+%     centroids_mat: position data from get_mouse_XY_pos
+%
+% 
+% 2015-03-01 Fori Wang
+
+    % find trial start and ends
+    [trial_frames,~]=find_start_end_of_trials(centroids_mat);
```

Comment:
This function should allow the user to specify a PlusMaze text file. If the behavior video and the text file are aligned, then the trial frame indices from the text file provide perfect knowledge into the trial boundaries. The function should try to recover trial frames through `find_start_end_of_trials` only if the plus maze text file is not provided.